### DISCUSSION TITLE
How to Animation

### DISCUSSION THREAD

**Discussion by sschueller** [healthchecks/healthchecks#1043] [General]
Hi, I just wanted to ask what tool you used to create the cool howto SVG animation on the website?

**Comment by cuu508** [comment]
I used pen and paper to make a sketch, Inkscape to create a static image, and wrote the animation code with a text editor.

**Reply by sschueller** [reply]
Thanks, looks great.

**Comment by strongmasternet** [comment]
Thanks for explanation
